client/blocks/inline-help/popover.jsx

```diff
@@ -96,6 +109,79 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	renderChecklistProgress = () => {
+		const { taskStatuses, siteSuffix, translate, sectionName } = this.props;
+
+		const isAtomicSite = get( this.props, 'selectedSite.options.is_automated_transfer' );
+		const isJetpackSite = get( this.props, 'selectedSite.jetpack' );
+		const inlineHelpButtonClasses = {
+			'checklist-count-notification': this.state.showChecklistNotification,
+			'inline-help__checklist-count': true,
+		};
+
+		const disallowedSections = [
+			'discover',
+			'reader',
+			'reader-activities',
+			'reader-list',
+			'reader-recommendations',
+			'reader-tags',
+			'checklist',
+		];
+
+		const totalTasks = tasks.length;
+		const numComplete = reduce(
+			tasks,
+			( count, { id, completed: taskComplete } ) =>
+				taskComplete || get( taskStatuses, [ id, 'completed' ] ) ? count + 1 : count,
+			0
+		);
+
+		//We have no tasks
+		if ( ! totalTasks ) {
+			return false;
+		}
+
+		//This is an Atomic or Jetpack site
+		if ( isAtomicSite || isJetpackSite ) {
+			return false;
+		}
+
+		//We're in the Reader area or the Checklist
+		if ( disallowedSections.indexOf( sectionName ) > -1 ) {
+			return false;
+		}
+
+		if ( numComplete === totalTasks ) {
+			// @TODO show something else?
+			return false;
+		}
+		const checklistLink = '/checklist' + siteSuffix;
+
+		return (
+			<div className="inline-help__checklist">
+				<Button
+					onClick={ this.checklistClicked }
+					href={ checklistLink }
+					className="inline-help__checklist-button"
+					borderless
+				>
+					<span className="inline-help__checklist-label">
+						{ translate( 'Continue Site Setup' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Continue your site setup:' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).